### PR TITLE
EZP-32198 - update Perso docs/add screenshots

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | EZP-32198 |
| Versions      | master |

### CODE REVIEW

File: docs/personalization/filters.md [hunk 1]
```diff
@@ -1,136 +1,144 @@
...
-the boost filter allows shifting (boosting) selected recommended items to the top positions of 
-the recommendation response. 
-This may happen if a selected attribute of the customer has the same value as the selected attribute 
-of the recommended item. 
-For example, we can show the news from the user's home country with higher priority than the rest.
+The category path parameter is a powerful tool. 
+A typical approach is to represent the default content in the navigation-based 
+structure of a website. 
+If you need to represent available items of a website in different dimensions (taxonomies), 
+you can do this out-of-the box, by enriching the `categorypath` information of an item.
```
Comment (DominikaK) [inline docs/personalization/filters.md:113]:
```suggestion
you can do this out of the box, by enriching the `categorypath` information of an item.
```

File: docs/personalization/filters.md [hunk 2]
```diff
@@ -1,136 +1,144 @@
...
 
-Assuming every item in the picture below has the attribute "country" and the user has the attribute 
-"country\_of\_origin", the boost filter can be configured in **Filter settings**, 
-which leads to boost recommendations for certain users:
+For example, the shop mentioned above sells both furniture and plants, and products
```
Comment (DominikaK) [inline docs/personalization/filters.md:115]: "shop mentioned above" - it either isn't mentioned, or too far away to refer to it

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/personalization/enabling_personalization.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+On the welcome screen, enter the following details in their respective fields:
+
+- A full name of the person responsible for accepting the terms and conditions
+- An email address to which you want the confirmation to be sent
+- An installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
+
+Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Submit**.
+Your request is sent to Ibexa, and you will receive a confirmation email in response.
+
+## Configuring mandator credentials
```
Comment (dabrt) [inline docs/personalization/enabling_personalization.md:22]:
```suggestion
## Configuring customer credentials
```

File: docs/personalization/personalization.md [hunk 4]
```diff
@@ -1,16 +1,30 @@
-# 1. Personalization
+# Introduction to personalization
 
-A cloud based personalization service leverages artificial intelligence and machine learning 
+A cloud-based personalization service leverages artificial intelligence and machine learning 
 technologies to deliver optimized customer experience. 
 You can capture user preferences and interests to offer personalized content.
```
Comment (dabrt) [inline docs/personalization/personalization.md:5]:
```suggestion
You can capture user preferences and interests to generate recommendations 
that can be used to offer personalized content.
```

File: docs/personalization/scenarios.md [hunk 5]
```diff
@@ -1,32 +1,80 @@
...
+Every recommendation request delivers only content of one output content type 
+(even if multiple types are selected in the interface below). 
+The output type is set during the recommendation request and must be covered by 
+the list of the supported content types in the requested scenario.
+
+![Basic scenario configuration](img/scenario_configuration.png "Basic scenario configuration")
+
+## Strategy configuration
+
+To modify the strategy, you drag model boxes between a list of all available models and the scenario
```
Comment (dabrt) [inline docs/personalization/scenarios.md:29]:
```suggestion
To modify the strategy, you drag model boxes between a list of all available models and the strategy 
```

File: docs/personalization/scenarios.md [hunk 6]
```diff
@@ -1,32 +1,80 @@
...
+(even if multiple types are selected in the interface below). 
+The output type is set during the recommendation request and must be covered by 
+the list of the supported content types in the requested scenario.
+
+![Basic scenario configuration](img/scenario_configuration.png "Basic scenario configuration")
+
+## Strategy configuration
+
+To modify the strategy, you drag model boxes between a list of all available models and the scenario 
+matrix on the right side.
```
Comment (dabrt) [inline docs/personalization/scenarios.md:30]:
```suggestion
board on the right side.
```

File: docs/personalization/scenarios.md [hunk 7]
```diff
@@ -1,32 +1,80 @@
...
+To avoid empty or insufficient recommendation results, add several models to a strategy.
+
+![Strategy configuration](img/scenario_configuration_strategy.png "Strategy configuration")
+
+You can arrange models within a strategy matrix by importance.
+Models from each category are used in parallel and strategy results contain an equally
```
Comment (dabrt) [inline docs/personalization/scenarios.md:36]:
```suggestion
Models from each level are used in parallel and strategy results contain an equally 
```

File: docs/personalization/scenarios.md [hunk 8]
```diff
@@ -1,32 +1,80 @@
...
+
+## Strategy configuration
+
+To modify the strategy, you drag model boxes between a list of all available models and the scenario 
+matrix on the right side.
+To avoid empty or insufficient recommendation results, add several models to a strategy.
+
+![Strategy configuration](img/scenario_configuration_strategy.png "Strategy configuration")
+
+You can arrange models within a strategy matrix by importance.
```
Comment (dabrt) [inline docs/personalization/scenarios.md:35]:
```suggestion
You can arrange models within a scenario board by importance.
```

File: docs/personalization/personalization.md [hunk 9]
```diff
@@ -1,16 +1,31 @@
...
-    Keep in mind that a product or item can be referred to as content and the buy event for non-paid 
-    publisher content is replaced by the consume event.
+    Both products or content items can be referred to as content and the BUY event can be understood as
+    the CONSUME event.
+
+Before you can use the personalization service, you must [enable it](enabling_personalization.md).
+After you do it, for the service to generate relevant recommendations, 
+you can [change the default configuration](perso_configuration.md), and then 
+you can [feed it with data](content_import.md), or wait until the service gathers 
+enough information about the [monitored content](content_types.md) and [events](event_types.md).
```
Comment (dabrt) [inline docs/personalization/personalization.md:23]:
```suggestion
enough information about the [monitored content](content_types.md) and events. 
```

File: docs/personalization/filters.md (hunk 2 above)
Comment (dabrt) [inline docs/personalization/filters.md:115]:
```suggestion
For example, in a shop that uses a category structure described in the preceding section to sell furniture
```

File: docs/personalization/filters.md [hunk 10]
```diff
@@ -1,136 +1,144 @@
...
+A typical approach is to represent the default content in the navigation-based 
+structure of a website. 
+If you need to represent available items of a website in different dimensions (taxonomies), 
+you can do this out-of-the box, by enriching the `categorypath` information of an item.
 
-Assuming every item in the picture below has the attribute "country" and the user has the attribute 
-"country\_of\_origin", the boost filter can be configured in **Filter settings**, 
-which leads to boost recommendations for certain users:
+For example, the shop mentioned above sells both furniture and plants, and products 
+are structured based on site navigation.
```
Comment (dabrt) [inline docs/personalization/filters.md:116]:
```suggestion
and plants, products can be structured based on site navigation. 
```

File: docs/personalization/recommendation_models.md [hunk 11]
```diff
@@ -1,226 +1,202 @@
...
+This model is not based on historical records but relies on the imported product catalog.
 
-#### Editor based
+#### Editor-based
 
-Products that are manually selected by a human. 
-A customer can replace an automatically generated recommendation with a predefined list. 
-It is best suited when the store administrator wants to add special offers or sell stock remains. 
-It could also be called "static recommendations".
+This model returns products from a list that you manually create if you have Edit privileges.
```
Comment (dabrt) [inline docs/personalization/recommendation_models.md:94]:
```suggestion
This model returns products from a list that you manually create if you have `Personalization/Edit` permissions. 
```

File: docs/personalization/scenarios.md [hunk 12]
```diff
@@ -1,32 +1,80 @@
...
-requested recommendations. 
-On the right side of the first rows there is a button to configure Category Filter usage in the recommendation request. 
+- A content type to be returned as recommendation 
+- A strategy (set of models) that is used for generating recommendations
+- A filter configuration
+
+If your [user Role](../site_organization/organizing_the_site.md#permissions) includes 
+the `Personalization/View` Policy, you can view the list of scenarios and preview their result.
+
+To modify the scenarios to suit your needs, you must have the `Personalization/Edit` permissions.
```
Comment (DominikaK) [inline docs/personalization/scenarios.md:13]:
```suggestion
To modify the scenarios to suit your needs, you must have the `Personalization/Edit` Policy.
```

**Review by DominikaK — APPROVED** [review]

**Pull request merged**